components/File/src/index.scss

```diff
@@ -0,0 +1,59 @@
+.denhaag-file {
+  display: flex;
+  color: inherit;
+  text-decoration: inherit;
+  border: var(--denhaag-file-border);
```

Comment:
To make sure our design tokens will also work in other platforms later we unfortunately cannot use these CSS specific shorthand properties. So for NL Design System we propose to split them up into logical properties: `border-width` `border-color` `border-style` (which might be a hardcoded `solid` if dashed and others won't make sense. 
b